layouts/README.md

```diff
@@ -1,8 +0,0 @@
-If you are using Jekyll, you may want to take advantage
-of its [layouts](https://jekyllrb.com/docs/step-by-step/04-layouts/)
-feature.
-
-We provide a file in this directory called archive.html
-that you can copy into your `_layouts` directory.  You can
-modify it as needed to add more branding/information to
-your archive.
```

Comment:
Is there a replacement for this when using the GHA deployment method?

If I understand the docs and the `action.yml` correctly, there's no way to include a `settings.py` file to make use of `settings.html_head_title` as a replacement.

Reply:
Ah, right, I didn't notice. But then again, in the current master, the `layouts/` dir doesn't get copied over either when executing the GHA.

I'm not entirely sure if others agree with my `html_head_title` implementation. They might prefer a file instead of a string. So waiting for others' opinion first.

Reply:
> But then again, in the current master, the layouts/ dir doesn't get copied over either when executing the GHA.

Doesn't it? We've used it to "style" the Julia archive a bit and it seems to work: [`_layouts/archive.html`](https://github.com/JuliaCommunity/zulip-archive/blob/master/_layouts/archive.html) and https://juliacommunity.github.io/zulip-archive/ (View Source)

Reply:
Ah, right, this line does the copying: https://github.com/zulip/zulip-archive/blob/d1e0bddf03c2ef0c85ebf58293eff2231bc85690/entrypoint.sh#L64.

Note to others: if `html_head_title` were an html file instead, it has a weird property that it cannot have a `</html>` tag, and so would be a malformed html file.

Reply:
I renamed `html_head_title` to `page_head`.

Reply:
Made a tracking issue for this request: #63.